test/fixtures/templates/bad/resources/sqs/retention_period.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Extension from `.yml` to [`.yaml`](https://travis-ci.com/github/aws-cloudformation/cfn-python-lint/jobs/367200676#L300) for this test:

https://github.com/aws-cloudformation/cfn-python-lint/blob/bc65cb3d160486cbc664664f752ed9399a4234fd/test/unit/rules/resources/test_retentionperiod.py#L27